### PR TITLE
[stable/sealed-secrets] Add Prometheus + Grafana monitoring resources

### PR DESCRIPTION
**Pull request by ghost**
#### What this PR does / why we need it:
This PR adds a couple of resources useful for monitoring sealed-secrets using Prometheus + Grafana.

- 1x ConfigMap that contains the `sealed-secrets-controller.json` Grafana Dashboard. This dashboard is loaded in by the Grafana sidecar. Read More:
  - https://johnharris.io/2019/03/dynamic-configuration-discovery-in-grafana/
  - https://github.com/bitnami-labs/sealed-secrets/blob/master/contrib/prometheus-mixin/dashboards/sealed-secrets-controller.json

- 1x PrometheusOperator ServiceMonitor that creates a target for Prometheus to scrape metrics from sealed-secrets controller

I've updated the README.md respectively

#### Special notes for your reviewer:

Thank you for taking the time time to review my pull request, please let me know if I need to make any more additional changes.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/sealed-secrets/templates/configmap-dashboards.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{{- if .Values.dashboards.create }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "sealed-secrets.fullname" . }}-controller-dashboard
+  namespace: {{ .Release.Namespace }}
+  labels:
+    grafana_dashboard: "1"
```
Comment (srueg) [inline stable/sealed-secrets/templates/configmap-dashboards.yaml:11]: Please add the possibility to define custom labels (like in the ServiceMonitor).

File: stable/sealed-secrets/dashboards/sealed-secrets-controller.json [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+            "2d",
+            "7d",
+            "30d"
+        ]
+    },
+    "timezone": "",
+    "title": "Sealed Secrets Controller",
+    "uid": "UuEtZCVWz",
+    "version": 2
+}
```
Comment (srueg) [inline stable/sealed-secrets/dashboards/sealed-secrets-controller.json]:
```suggestion
}

```

**Review by srueg — CHANGES_REQUESTED** [review]

**Comment by KadenLNelson** [comment]
@srueg Addressed your comments in the latest commit. Sorry about the force-push, I didn't DCO sign a commit.

**Comment by KadenLNelson** [comment]
@olib963 May you please review the pull request?

**Review by olib963 — APPROVED** [review]
Sorry this took a while, it seems I don't get notified when someone does `/assign` but only when they `@` :confused:.

I can't comment much on the grafana/prometheus implementation myself, if you want me to really thoroughly review I will try and spin it up locally to see what happens?

Or if you want the CI to test it I believe you can add a new custom values file for testing.

It may be worth noting that it seems there is a desire to move the ownership of this chart elsewhere (https://github.com/bitnami/charts/pull/4017), I am sure whoever did that in the end would pull over these changes too, just wanted to make you aware in case you were not already.

**Pull request merged**